avocado/plugins/nrun.py

```diff
@@ -118,20 +138,12 @@ def spawn_task(self, task):
             stdout=asyncio.subprocess.PIPE,
             stderr=asyncio.subprocess.PIPE)
 
-    def check_tasks_requirements(self, tasks):
-        result = []
-        for task in tasks:
-            runner = pick_runner(task, self.KNOWN_EXTERNAL_RUNNERS)
-            if runner:
-                result.append(task)
-            else:
-                LOG_UI.warning('Task will not be run due to missing requirements: %s', task)
-        return result
-
     def run(self, config):
         resolutions = resolver.resolve(config.get('references'))
         tasks = job.resolutions_to_tasks(resolutions, config)
-        self.pending_tasks = self.check_tasks_requirements(tasks)  # pylint: disable=W0201
+        self.pending_tasks = check_tasks_requirements(   # pylint: disable=W0201
```

Comment:
Should we remove this pylint disable?

Looks like we need to do a `self.pending_tasks = something`  on `__init__`.